examples/time-filter-webhook.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Sensor
+metadata:
+  name: webhook-example
+  labels:
+    sensors.argoproj.io/controller-instanceid: axis
+spec:
+  signals:
+    - name: webhook
+      webhook:
+        endpoint: /app
+        method: POST
+      filters:
+        time:
+          start: "2016-05-10T15:04:05Z07:00"
```

Comment:
Does this mean that the trigger will be fired only if the signal is fired between the `start` and `stop` time?